examples/nodejs_etcd_operator/service-binding-request.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps.openshift.io/v1alpha1
```

Comment:
Do we need to keep those extra `yaml` files around? Would having only the contents in the example markdown file be sufficient?

Reply:
having everything in `readme` does make sense. I can remove all unwanted yamls